op-node/rollup/engine/payload_success.go

```diff
@@ -30,11 +34,9 @@ func (eq *EngDeriver) onPayloadSuccess(ev PayloadSuccessEvent) {
 		})
 	}
 
-	payload := ev.Envelope.ExecutionPayload
-	eq.log.Info("Inserted block", "hash", payload.BlockHash, "number", uint64(payload.BlockNumber),
-		"state_root", payload.StateRoot, "timestamp", uint64(payload.Timestamp), "parent", payload.ParentHash,
-		"prev_randao", payload.PrevRandao, "fee_recipient", payload.FeeRecipient,
-		"txs", len(payload.Transactions), "concluding", ev.Concluding, "derived_from", ev.DerivedFrom)
-
-	eq.emitter.Emit(TryUpdateEngineEvent{})
+	eq.emitter.Emit(TryUpdateEngineEvent{
+		BuildStarted:  ev.BuildStarted,
+		InsertStarted: ev.InsertStarted,
+		Envelope:      ev.Envelope,
```

Comment:
`Envelope` is included in the `TryUpdateEngineEvent` because it contains the following block info that I wanted to include in the log:
```
BlockNumber
BlockHash
GasUsed
StateRoot
Timestamp
ParentHash
PrevRandao
FeeRecipient
Transactions (I am logging len(payload.Transactions)
```
I would be open to not logging some of that info (and therefore not needing to pass it to the `TryUpdateEngineEvent`, but at a minimum I would like to access the following fields:
```
BlockNumber
BlockHash
GasUsed
```